quarkus-extension/engine/deployment/src/test/java/org/camunda/bpm/quarkus/engine/test/config/CamundaEngineConfigurationConfigTest.java

```diff
@@ -65,7 +65,7 @@ public void shouldApplyProcessEngineConfigurationProperties() {
     // then
     assertThat(configuration.isCmmnEnabled()).isEqualTo(false);
     assertThat(configuration.isDmnEnabled()).isEqualTo(false);
-    assertThat(configuration.getHistory()).isEqualTo("none");
+    assertThat(configuration.getHistory()).isEqualTo("full");
```

Comment:
This should be `none` since we set it in the `mixed-application.properties` file which has a higher priority.